run-acorn.pl

```diff
@@ -8,31 +8,22 @@
 # April 2000
 
 $uid = "mta";
-#$work_dir = "/proj/ascwww/AXAF/extra/science/cgi-gen/mta/Snap";
 $work_dir = "/data/mta4/www/Snapshot";
 $pid_file = "$work_dir/racorn.pid";
-#@acorn = qw(/home/ascds/DS.release/bin/acorn /proj/cm/Release/install.DS7.6.9/bin/acorn /data/mta2/pallen/acorn-1.33/acorn /home/swolk/acorn/src1-3/acorn);
 @acorn = qw(/home/ascds/DS.release/bin/acorn);
-#@acorn = qw(/export/acis-flight/primary/acorn-1.33/acorn);
-#@acorn = qw(/home/acisdude/real-time/back-up/acorn-1.33/acorn);
-#@acorn = qw(/home/mta/ACORN/acorn1.52);
 $acorn_exe = (-e $acorn[0])? $acorn[0] : $acorn[1];
-#$UDP_port = "11111"; # eno's acorn feed
 $UDP_port = "11112"; # c3po-v
-#$UDP_port = "11113"; # temp feed from forbin bds 09/11/03
 $msids = "$work_dir/chandra-msids.list";
-#$msids = "$work_dir/msids.list";
 $filesize = 500;
 
 # set environment variables for acorn
 
 @mta_data = qw(/home/ascds/DS.release/config/mta/data /data/mta2/pallen/acorn-1.3/groups /home/swolk/acorn/groups);
 $ENV{ASCDS_CONFIG_MTA_DATA} = (-e $mta_data[0])? $mta_data[0] : $mta_data[1]; 
+@ipcl = qw(/home/ascds/DS.release/config/tp_template/P016 /data/mta4/www/Snapshot/P011)
```

Comment:
Are we maintaining an array of IPCL format options to remain backwards compatible? 

Also a diff of the old P011 template in the latest DS.release/config/tp_template/P011/tpp.txt file and the MTA copied version Snapshot/P011/tpp.txt file show a small discrepancy on data values for the "2DETART" and "2SHLDART" MSIDS. Will this cause an issue if we try to run acorn with the old P011 format?

Reply:
Double checked. Regardless of discrepancy, we should still prioritize running the processing template with the P016 release or later as exhibited by the previous issues of Dumps-OTG processing missing data due to not using the correct template.